### ISSUE TITLE
Too few arguments to function Kiener\MolliePayments\Service\CartService::__construct()

### ISSUE THREAD

**Issue by MelvinAchterhuis** [mollie/Shopware6#177]
![image](https://user-images.githubusercontent.com/26538915/128824765-dfec2498-86f4-4e98-801a-bc4d8789d90b.png)

In MolliePayments/src/Resources/config/services.xml line 46 only 1 argument is passed:

```
<!-- Services -->
<service id="Kiener\MolliePayments\Service\CartService" class="Kiener\MolliePayments\Service\CartService">
    <argument type="service" id="Shopware\Core\Checkout\Cart\SalesChannel\CartService"/>
</service>
```

This issue currently throws errors in browser console:

![Screenshot 2021-08-10 at 09 20 51](https://user-images.githubusercontent.com/26538915/128825128-73f91e6a-4283-4451-9d8f-02b30ce5d371.png)

**Comment by schmitzcarsten** [comment]
we are already aware of this and will provide a fix for this soon. At the meantime you could add the second argument in the services.xml manually

**Comment by schmitzcarsten** [comment]
fixed with MOL-540 commit. will be in next release

**Comment by schmitzcarsten** [comment]
fixed in v1.5.3
